templates/components/levels.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %} {% block content %}
+
+<script>
+  $(document).ready(() => {
+    $('#levelsTable').DataTable();
+
+    // show quick add modal if quick-modal to true
+    const url = new URL(window.location.href);
+    if (url.searchParams.get('quick-action')) {
+      $('#addLevelModal').modal('show');
+    }
+  });
+</script>
+
+<div class="container">
+  {% block breadcrumbs %}
+  <ul class="breadcrumb">
+    <li><a href="{% url 'index' %}">My Dashboard</a></li>
+    <li><a href="/indicators/home/0/0/0/">Indicators</a></li>
+    <li class="active">Levels</li>
+  </ul>
+  {% endblock %}
+
+  <!-- Sub navigation -->
+  <div class="sub-navigation">
+    <div class="sub-navigation-header">
+      <h4 class="page-title">
+        Level List
+      </h4>
+    </div>
+    <div class="sub-navigation-actions">
+      <!-- sample action buttons -->
+      <div class="sub-nav-item">
+        <div class="btn-group" role="group" aria-label="">
+          <a role="button" class="btn btn-sm btn-default disabled">
+            <i class="fa fa-list"></i> List
+          </a>
+
+          <a role="button" class="btn btn-sm btn-default" href="">
+            <i class="fa fa-sitemap"></i> Tree
+          </a>
+        </div>
+      </div>
+
+      <div class="sub-nav-item">
+        <div class="btn-group" role="group" aria-label="">
+          <button class="btn btn-sm btn-primary" data-toggle="modal" data-target="#addLevelModal">
+            <i class="fa fa-plus"></i> Level
```

Comment:
Hello @kenneth051 , Good working on this, on click though this modal is not raised.

Reply:
@kelvinndmo Yes, That will come in another PR. I have two tickets I am working on.

Reply:
Oh awesome, thanks for the head up